sechub-web-ui-solution/docker/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env sh
+# SPDX-License-Identifier: MIT
+
+debug () {
+    while true
+    do
+	    echo "Press [CTRL+C] to stop.."
+	    sleep 120
+    done
+}
+
+server () {
```

Comment:
I would remove this whole function and add the commands to the end of the script.
Then it's more obvious what's getting started. :-)
```suggestion
```